packages/js/src/structured-data-blocks/faq/components/Question.js

```diff
@@ -336,16 +336,18 @@ export default class Question extends Component {
 		return (
 			<div className="schema-faq-section" key={ id }>
 				<RichText
+					identifier={ id + "-question" }
 					className="schema-faq-question"
 					tagName="p"
 					key={ id + "-question" }
 					value={ question }
 					onChange={ this.onChangeQuestion }
 					unstableOnFocus={ this.onFocusQuestion }
 					placeholder={ __( "Enter a question", "wordpress-seo" ) }
-					formattingControls={ [ "italic", "strikethrough", "link" ] }
+					allowedFormats={ [ "core/italic", "core/strikethrough", "core/link", "core/annotation" ] }
```

Comment:
Here, I changed the RichText prop `formattingControls` to `allowedFormats`. This is because `formattingControls` is depecrated since 2019. `allowedFormats` is the new prop, see [this doc](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/rich-text/README.md#allowedformats-array).

In addition, I added "core/annotation" format to allow annotations to be applied in the question part of the block.

Reply:
Awesome find! 👍

Reply:
Also noteworthy that removing this property, as we have for the answer below, allows all possible formatting (including core/annotation). That's why you initially only had annotation working for answers.

Reply:
The same happened for How-To steps. Initially only the step description would work, but not the title. Changing to `allowedFormats` and adding the "core/annotation" format solved the issue!